### PR TITLE
New issue forms

### PR DESCRIPTION
**Pull request by HakonRydland**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information: New issue forms as per dev meeting 03/07/2021
    

### Please specify which Issue this PR Resolves.
none

### Please verify the following and ensure all checks are completed.
None applicable
1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/GameBalanceForm.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Game balance
+description: Sugguest changes to the ballance of the mission.
```
Comment (Lazejun) [inline .github/ISSUE_TEMPLATE/GameBalanceForm.yml:2]: You found an extra L here
```suggestion
description: Sugguest changes to the balance of the mission.

```

File: .github/ISSUE_TEMPLATE/ChangeRequestForm.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    id: solution
+    attributes:
+      label: Describe the solution you'd like.
+      description: A clear and concise description of how and what you'd like to see changed.
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternative soloutions
```
Comment (Lazejun) [inline .github/ISSUE_TEMPLATE/ChangeRequestForm.yml:23]: You have 3 times "Alternative soloutions", where you found an Extra O in solutions

File: .github/ISSUE_TEMPLATE/GameBalanceForm.yml (hunk 1 above)
Comment (HakonRydland) [reply .github/ISSUE_TEMPLATE/GameBalanceForm.yml:2]: balance is actually correct, but there is an "u" to many in Suggest 😄

Comment (Lazejun) [reply .github/ISSUE_TEMPLATE/GameBalanceForm.yml:2]: :P

File: .github/ISSUE_TEMPLATE/BugReportForm.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      label: Describe the bug
+      description: Also tell us, what did you expect to happen?
+      placeholder: A bug happened! I think this is what should have happened.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: How to reproduce
+      description: How can wwe reproduce this bug?
```
Comment (Lazejun) [inline .github/ISSUE_TEMPLATE/BugReportForm.yml:18]:
```suggestion
      description: How can we reproduce this bug?

```

**Review by Lazejun — APPROVED** [review]
✨🌟✨ Shiny ✨🌟✨

Looks good to me

**Pull request merged**